app/src/main/res/values/strings.xml

```diff
@@ -429,7 +429,7 @@
   <string name="linkidwithpatient_button_yes">Yes</string>
   <string name="linkidwithpatient_button_back">Back</string>
   <string name="linkidwithpatient_text">Add BP Passport to</string>
-  <string name="linkidwithpatient_patient_text">Patient %1$s?</string>
```

Comment:
Two strings are still left that have `Patient` into it. Can you remove those as well?

Reply:
Ah, it looks like a couple of translations PRs got merged in since I opened this PR. I am gonna wait until the rest of them get merged in to avoid any unnecessary issues again.